saleae/saleae.py

```diff
@@ -439,111 +447,132 @@ def load_from_file(self, file_path_on_target_machine):
 	def close_all_tabs(self):
 		self._cmd('CLOSE_ALL_TABS')
 
-	def export_data(self,
-			file_path_on_target_machine,
-			digital_channels=None,
-			analog_channels=None,
-			analog_format="voltage",
-			time_span=None,				# 'None-->all_time, [x.x, y.y]-->time_span'
-			format="csv",				# 'csv, bin, vcd, matlab'
-			csv_column_headers=True,
-			csv_delimeter='comma',		# 'comma' or 'tab'
-			csv_timestamp='time_stamp',	# 'time_stamp, sample_number'
-			csv_combined=True,			# 'combined' else 'separate'
-			csv_row_per_change=True,	# 'row_per_change' else 'row_per_sample'
-			csv_number_format='hex',	# dec, hex, bin, ascii
-			bin_per_change=True,		# 'on_change' else 'each_sample'
-			bin_word_size='8'			# 8, 16, 32, 64
-			):
-		# export_data, C:\temp_file, digital_channels, 0, 1, analog_channels, 1, voltage, all_time, adc, csv, headers, comma, time_stamp, separate, row_per_change, Dec
-		# export_data, C:\temp_file, all_channels, time_span, 0.2, 0.4, vcd
-		# export_data, C:\temp_file, analog_channels, 0, 1, 2, adc, all_time, matlab
+	def _export_data_analog_binary(self, analog_format='voltage'):
+		'''Binary analog: [VOLTAGE|ADC]'''
+
+		# Do argument verification
+		if analog_format.lower() not in ['voltage', 'adc']: raise ValueError('bad binary analog format')
```

Comment:
Stylistically, would you mind pushing the `raise` to the next line to match the rest of the file? I wouldn't ask if there wasn't already another change from above.